### DISCUSSION TITLE
扫图指南

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#37] [Announcements]
# 扫图指南

本文旨在提供从图像扫描到图像处理的完整流程参考（没有完全遵循须知部分的指导），在实践中，你可根据自己的实际需求省略步骤、调整参数等。

https://zexwoo.blog/posts/tutorials/artwork-scanning/

**Comment by KenCAH** [comment]
博主大大，我想问一下如果没有扫描仪但是有高素质的微单及微距镜头的话，可否用作歌词本等的数字化？

**Reply by ZexWoo** [reply]
当然是可以的，有胜于无。看起来清楚整齐，也不见得比扫描仪差。

不过一些后处理可能会不一样，你可以尝试着摸索一下。
